src/modules/hip/hip_advanced_augmentation.cpp

```diff
@@ -0,0 +1,924 @@
+
+RppStatus
+lut_hip_batch_tensor(Rpp8u* srcPtr, Rpp8u* dstPtr, Rpp8u* lut, rpp::Handle &handle, RPPTensorFunctionMetaData &tensor_info)
+{
+    int in_plnpkdind = getplnpkdind(tensor_info._in_format), out_plnpkdind = getplnpkdind(tensor_info._out_format);
+    InitHandle *handle_obj = handle.GetInitHandle();
+    Rpp32u max_height, max_width;
+    max_size(handle_obj->mem.mgpu.csrcSize.height, handle_obj->mem.mgpu.csrcSize.width, handle.GetBatchSize(), &max_height, &max_width);
+
+#if defined (HIPRTC)
+
+    std::vector<size_t> vld{16, 16, 1};
+    std::vector<size_t> vgd{max_width, max_height, handle.GetBatchSize()};
+    std::string kernel_file = "look_up_table.cpp";
+    std::string kernel_name = "look_up_table_batch_tensor";
+    get_kernel_name(kernel_name, tensor_info);
+
+    handle.AddKernel("", "", kernel_file, kernel_name, vld, vgd, "")(srcPtr,
+                                                                     dstPtr,
+                                                                     lut,
+                                                                     handle_obj->mem.mgpu.srcSize.height,
+                                                                     handle_obj->mem.mgpu.srcSize.width,
+                                                                     handle_obj->mem.mgpu.maxSrcSize.width,
+                                                                     handle_obj->mem.mgpu.srcBatchIndex,
+                                                                     tensor_info._in_channels,
+                                                                     handle_obj->mem.mgpu.inc,
+                                                                     handle_obj->mem.mgpu.dstInc,
+                                                                     in_plnpkdind,
+                                                                     out_plnpkdind);
+
+    #elif defined(STATIC)
+
+            hip_exec_lut_batch_tensor(srcPtr, dstPtr, lut, handle, tensor_info, in_plnpkdind, out_plnpkdind, max_height, max_width);
+    #endif
+    return RPP_SUCCESS;
+}
+
+RppStatus
+lut_hip_batch_tensor_fp16(Rpp16f *srcPtr1, Rpp16f *srcPtr2, Rpp16f *dstPtr, Rpp8u* lut, rpp::Handle &handle, RPPTensorFunctionMetaData &tensor_info)
+{
+//     int in_plnpkdind = getplnpkditensor_info._in_format), out_plnpkdind = getplnpkdind(tensor_info._out_format);
+//     InitHandle *handle_obj = handle.GetInitHandle();
+//     Rpp32u max_height, max_width;
+//     max_size(handle_obj->mem.mgpu.csrcSize.height, handle_obj->mem.mgpu.csrcSize.width, handle.GetBatchSize(), &max_height, &max_width);
+// #if defined (HIPRTC)
+// #elif defined(STATIC)
+//      hip_exec_lut_batch_tensor_fp16(srcPtr1, srcPtr2, dstPtr, handle, in_plnpkdind, out_plnpkdind, max_height, max_width);
+// #endif
+    return RPP_SUCCESS;
+}
+
+RppStatus
+lut_hip_batch_tensor_fp32(Rpp32f *srcPtr, Rpp32f *dstPtr, Rpp32f* lut, rpp::Handle &handle, RPPTensorFunctionMetaData &tensor_info)
```

Comment:
Just for lut, remove this lut_hip_batch_tensor_fp32 function too

Reply:
done